bitcoin/src/network/mod.rs

```diff
@@ -386,11 +386,11 @@ mod tests {
     #[cfg(feature = "serde")]
     fn serde_roundtrip() {
         use Network::*;
-        let tests = vec![
+        let tests = [
             (Bitcoin, "bitcoin"),
             (Testnet, "testnet"),
             (Signet, "signet"),
-            (Regtest, "regtest"),
+            (Regtest, "regtest")
```

Comment:
In 1518efc9a16dc8932821399efdc8349c73602c1f

Can you put this comma back?

Reply:
Sure